converters/fromZigbee.js

```diff
@@ -5218,7 +5218,15 @@ const converters = {
         type: 'commandStatusChangeNotification',
         convert: (model, msg, publish, options, meta) => {
             const zoneStatus = msg.data.zonestatus;
-            return {occupancy: (zoneStatus & 1<<2) > 0};
+            return {occupancy: (zoneStatus & 1) > 0};
```

Comment:
`ZB003X_tamper` can be merged with this: `return {occupancy: (zoneStatus & 1) > 0, tamper: (zoneStatus & 4) > 0};`

Reply:
Thanks, complete lack of experience here :). Let me try, and thanks for helping!!!

Reply:
I think I did it. Thanks again! The main function still is occupancy, it that a problem?

Reply:
Running some checks in my local instance before requesting new review!

Reply:
Done, everything working as expected!